docs/source/conf.py

```diff
@@ -27,16 +27,10 @@
 import aiida
```

Comment:
I believe you could also remove the addition of aiida to the path, above this, now if you want?

Reply:
Done